### PR TITLE
File system based indexers return record display name

### PR DESCRIPTION
**Pull request by six5532one**
This PR updates file system based indexers to return each file's path as its display name.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-## 0.2.1-dev2
+## 0.2.2
```
Comment (six5532one) [inline CHANGELOG.md:1]: I changed the version from a dev version to a release version and rolled the enhancements from the latest dev version into this new release version. Is that ok?

Comment (rbiseck3) [reply CHANGELOG.md:1]: Can we keep this dev so I can coordinate when the next release goes out?

File: unstructured_ingest/v2/processes/connectors/fsspec/fsspec.py [hunk 2]
```diff
@@ -166,6 +166,7 @@ def run(self, **kwargs: Any) -> Generator[FileData, None, None]:
 
             additional_metadata = self.sterilize_info(file_data=file_data)
             additional_metadata["original_file_path"] = file_path
+            additional_metadata["record_display_name"] = file_path
```
Comment (rbiseck3) [inline unstructured_ingest/v2/processes/connectors/fsspec/fsspec.py:169]: We should make this a concrete field on the FileData class, set to an optional string defaulted to None.

**Review by rbiseck3 — CHANGES_REQUESTED** [review]

File: unstructured_ingest/v2/processes/connectors/fsspec/fsspec.py (hunk 2 above)
Comment (six5532one) [reply unstructured_ingest/v2/processes/connectors/fsspec/fsspec.py:169]: @rbiseck3 why is this preferred?

Comment (rbiseck3) [reply unstructured_ingest/v2/processes/connectors/fsspec/fsspec.py:169]: It’ll provided a common interface that can be taken advantage of downstream from this. There’s no contractual guarantee on the data that lives in the additional metadata blob.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,14 @@
+## 0.2.1-dev3
```
Comment (rbiseck3) [inline CHANGELOG.md:1]: This should replace the one below rather then add a new one.

File: unstructured_ingest/v2/interfaces/file_data.py [hunk 4]
```diff
@@ -43,6 +43,7 @@ class FileData(DataClassJsonMixin):
     additional_metadata: dict[str, Any] = field(default_factory=dict)
     reprocess: bool = False
     local_download_path: Optional[str] = None
+    record_display_name: Optional[str] = None
```
Comment (rbiseck3) [inline unstructured_ingest/v2/interfaces/file_data.py:46]: Would this be more succinct? 
```suggestion
    display_name: Optional[str] = None
```

**Review by rbiseck3 — APPROVED** [review]

**Pull request merged**